common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -310,6 +316,21 @@ def _verify_cohort_settings(
                     self.cohort_management_page.get_cohort_associated_assignment_type()
                 )
 
+    def _create_csv_file(self, filename, csv_text_as_lists):
+        """
+        Create a csv file with the provided list of lists.
+
+        :param filename: this is the name that will be used for the csv file. Its location will
+         be under the test upload data directory
+        :param csv_text_as_lists: provide the contents of the csv file int he form of a list of lists
+        """
+        filename = self.instructor_dashboard_page.get_asset_path(filename)
+        with open(filename, 'w+') as csv_file:
+            writer = csv.writer(csv_file, quoting=csv.QUOTE_ALL)
```

Comment:
Nice! thanks for doing this @benpatterson!

Could you use the `unicodecsv` library and add a testcase ? We should make sure we can handle CSVs with utf-8-encoded unicode content.